wiki/Music_theory/Layer/en.md

```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - musical layer
+  - sound layer
+---
+
+# Layer
+
+A **musical layer** is a part of the song which contains features with a common aspect between them. A significant part of [mapping](/wiki/Beatmapping) in every [game mode](/wiki/Game_mode) revolves around following different kinds of layers (percussion, vocals, sound patterns) consistently, and switching between them for the purpose of emphasis and contrast.
+
+## Description
+
+Typically, a single layer in a musical piece represents a group of instruments of the same family or [range](https://en.wikipedia.org/wiki/Range_(music)), or — alternatively — voices of the [same type](https://en.wikipedia.org/wiki/Voice_type). Since layers are abstract, they can be limited to just a single instrument or kind of sound, depending on the context they are discussed in. Other criteria for layer subdivision include [rhythm](https://en.wikipedia.org/wiki/Rhythm) patterns, [melody](https://en.wikipedia.org/wiki/Polyphony), [harmony](https://en.wikipedia.org/wiki/Harmony), or [pitch](https://en.wikipedia.org/wiki/Pitch_(music)).
+
+## Mapping
+
+In osu! as a game, song layers are represented through different [patterns](/wiki/Beatmap/Pattern) and sections of the beatmap. While picking a layer to follow is usually up to a beatmap author's discretion, the choices are usually the most obvious and recognisable ones:
```

Comment:
i'm pretty sure it's more like the rhythm choice depends on which layer is followed

the patterns may or may not relate to layers or specific instruments, but the logic of when to use which pattern can depend on way more things

Reply:
yes. I wanted to explain that the musical layers translate to the hit objects (well, isn't that evident), but that seemed to be too simple